README.md

```diff
@@ -0,0 +1,580 @@
+# Xiana framework
+
+Xiana is a lightweight web-application framework written in Clojure LISP, its goal is to be simple, blasting fast, and
```

Comment:
"blazingly fast" is the usual idiom. Just "fast" might be better, depending on what style we want to go for. Sometimes its good to just drop as many adjectives/adverbs as possible, especially if they might be seen as hyperbole.